src/api_routes_http.rs

```diff
@@ -263,6 +264,7 @@ pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimitCell) {
             web::post().to(mark_person_mention_as_read),
           )
           .route("/replies", web::get().to(list_replies))
+          .route("/followed_posts", web::get().to(list_followed_posts))
```

Comment:
This needs endpoints for adding and removing specific communities to your notifs. I imagine we'd want new posts or new comments.

Not sure why followed posts would be necessary. Notifs should be sent whenever there's a `CreatePost` / `CreateComment` action.

Reply:
The reason I didn't take an action based approach is that there isn't some notifications table that I can write to whenever an action happens, instead the notifications are a collection of endpoints that the UI reads from whenever the user goes to that screen, so this seemed like the natural approach to fit the rest. Do you have any other idea about whenever a `CreatePost` or `CreateComment` action is received?

Reply:
`crates/api_crud/src/comment/create.rs`

`crates/api_crud/src/post/create.rs`

Check out the `send_local_notifs` function and trace down where its used, because this would live alongside or withing that.